docs/content/plugins/mongodb-docker.md

```diff
@@ -0,0 +1,40 @@
+---
+title: MongoDB Docker Storage Plugin
+description: A built-in plugin that stores Porter's data in a container running MongoDB.
+---
+
+The MongoDB Docker Storage plugin is built-in to Porter and is the default
+storage plugin. This plugin is suitable for development and test but should not
+be used in production.
+
+The plugin runs a MongoDB server in a container, storing its data on a separate
+volume. The container is named `porter-mongodb-docker-plugin` and the volume is
+named `porter-mongodb-docker-plugin-data`. The plugin leaves the container
+running in-between Porter commands for performance reasons. It is safe to stop
+or remove the container. Removing the volume will result in data loss.
+
+## Plugin Configuration
+
+To use the mongodb plugin, add the following config to porter's config file
+(default location: `~/.porter/config.toml`). Replace `conn_str` with the
```

Comment:
`Replace conn_str with the connection string for your MongoDB server.` should be removed, right?

Reply:
oops! thanks